### PR TITLE
Enhancement/new player select room

### PR DESCRIPTION
**Pull request by ChrisKrane**
Image_Sequence_004_0000

### CODE REVIEW

File: Assets/Scripts/PlayerSelect/PlayerEyeManager.cs [hunk 1]
```diff
@@ -13,14 +14,21 @@ public void Start()
     {
         // Find eye material
         SkinnedMeshRenderer eyeRenderer = transform.GetChild(0).GetComponent<SkinnedMeshRenderer>();
-        eyeMaterial = eyeRenderer.materials[1];
-        resetEyePosition();
+        Material[] playerMaterials = eyeRenderer.materials;
+        foreach (Material playerMaterial in playerMaterials)
+        {
+            if (playerMaterial.name == "PlayerEyesMaterial (Instance)") {
```
Comment (toberge) [inline Assets/Scripts/PlayerSelect/PlayerEyeManager.cs:20]: Could be accomplished with a .Single(m => m.name = "blah blah")

**Review by toberge — CHANGES_REQUESTED** [review]
Looks good!

Though, where did the cards on the table go? I felt they were a nice detail.

Also, could you make the table material toon shaded now? It seems to work quite well  with the current design:  
image

File: Assets/Scripts/PlayerSelect/PlayerEyeManager.cs (hunk 1 above)
Comment (ChrisKrane) [reply Assets/Scripts/PlayerSelect/PlayerEyeManager.cs]: Fixed, and re-added cards on table. I had completely overlooked them when reimporting the new model. Thank you!

**Review by Fueredoriku — APPROVED** [review]
looks great! 😁👍

**Review by toberge — APPROVED** [review]
Very nice!

**Pull request merged**